### DISCUSSION TITLE
Confused about docker image and livereload

### DISCUSSION THREAD

**Discussion by pvilas** [squidfunk/mkdocs-material#2835] [General]
Hi, 
I'm using the latest squidfunk/mkdocs-material on OSx. The reload of the documents seems not to work using VSCode or Sublime, but is working using vim!!!
Has anyone any clue about this behaviour?
Regards.

**Comment by squidfunk** [comment]
Yes. This is a regression of MkDocs 1.2.1. It ~~will~~ should be fixed as soon as 1.2.2 is out.

Related: https://github.com/mkdocs/mkdocs/issues/2491

**Reply by pvilas** [reply]
I see. Thanks!
